### ISSUE TITLE
NPOI does not work with .net standard

### ISSUE THREAD

**Issue by msevestre** [Open-Systems-Pharmacology/OSPSuite.Core#1288]
All right...so after spending 3 hours trying to understand why we could import from R, I realized that we have always used the .NET 4.5 version of NPOI and not the .net standard. Trying to use this version results in different error messages 

such as 
![image](https://user-images.githubusercontent.com/1041237/131735626-059177c2-a782-4d40-bdea-f38b32e400f4.png)

which is in fact

Could not load file or assembly 'System.Configuration.ConfigurationManager, Version=4.0.1.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies. The system cannot find the file specified.

We need to create a console application (.NET core) that try to load a file so that we can reproduce the issue in .NET. I can reproduce it by copying stuff by hand but it would be much easier to have an exe to reproduce the error

In short, at the moment, import of observed data will not work under Linux. We are probably simply missing a dependencies but...which one

and `System.Configuration.ConfigurationManager, Version=4.0.1.0` does not exist :)

**Comment by PavelBal** [comment]
Obsolete.
